### DISCUSSION TITLE
When downloading file ends up larger then original

### DISCUSSION THREAD

**Discussion by ctphoto** [iterate-ch/cyberduck#14687] [Q&A] [answered]
This does not happen every time, so hard to troubleshoot. When I download a TIFF, let's say an 845MB file, it downloads it and the stalls and the download keeps growing, up to 1.5GB. Then the file can not be opened in PhotoShop, errors out. If I try downloading it again it downloads 845MB and finishes and I can open the file.

The FTP server is a Windows 2000 PC running xFlight FTP software. I am on Mac OS 13.3.1 using CyberDuck 8.5.9

Any ideas?

**Comment by tatze96** [comment]
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), re-create the described behavior, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-14687 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Comment by tatze96** [comment, marked as answer]
Download issues with FTP connections where the transfer progress exceeds the maximum percentage and size are often caused by the segmented download setting. The *Segmented downloads with multiple connections per file* setting can be deactivated within the Cyberduck preferences *Transfers* tab.

**Comment by dkocher** [comment]
Duplicate for #13374.
